en/FlyView/FlyView.md

```diff
@@ -1,79 +1,375 @@
+If you want to resume the mission, confirm this and the mission will start from the last waypoint traveled through. 
+Once the mission is rebuilt you will be presented with another Resume Mission slide which allows you to review the rebuilt mission before starting it again.
+Confirm this *Resume Mission* slider to continue on with the mission.
+
+![Resume Mission](../../assets/fly/resume_mission.jpg)
+
+##### Note: How Resume Mission Rebuilding Works
+
+A mission cannot simply resume from the last mission item the vehicle executed because there may be multiple items at the last waypoint that affect the next stage of the mission (e.g. speed commands or camera control commands).
+Skipping any mission items at the last waypoint may prevent the rest of the mission running correctly.
+
+Therefore *QGroundControl* rebuilds the mission looking backwards from the last mission item flown, and automatically appends relevant commands to the front of the mission. 
+By doing this the state of the mission prior to the resume point is restored.
+
+The following mission commands are the ones scanned for:
+* [MAV_CMD_DO_CONTROL_VIDEO](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_CONTROL_VIDEO)
+* [MAV_CMD_DO_SET_ROI](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_ROI)
+* [MAV_CMD_DO_DIGICAM_CONFIGURE](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_DIGICAM_CONFIGURE)
+* [MAV_CMD_DO_DIGICAM_CONTROL](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_DIGICAM_CONTROL)
+* [MAV_CMD_DO_MOUNT_CONFIGURE](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_MOUNT_CONFIGURE)
+* [MAV_CMD_DO_MOUNT_CONTROL](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_MOUNT_CONTROL)
+* [MAV_CMD_DO_SET_CAM_TRIGG_DIST](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_CAM_TRIGG_DIST)
+* [MAV_CMD_DO_FENCE_ENABLE](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_FENCE_ENABLE)
+* [MAV_CMD_IMAGE_START_CAPTURE](https://mavlink.io/en/messages/common.html#MAV_CMD_IMAGE_START_CAPTURE)
+* [MAV_CMD_IMAGE_STOP_CAPTURE](https://mavlink.io/en/messages/common.html#MAV_CMD_IMAGE_STOP_CAPTURE)
+* [MAV_CMD_VIDEO_START_CAPTURE](https://mavlink.io/en/messages/common.html#MAV_CMD_VIDEO_START_CAPTURE)
+* [MAV_CMD_VIDEO_STOP_CAPTURE](https://mavlink.io/en/messages/common.html#MAV_CMD_VIDEO_STOP_CAPTURE)
+* [MAV_CMD_DO_CHANGE_SPEED](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_CHANGE_SPEED)
+* [MAV_CMD_NAV_TAKEOFF](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_TAKEOFF)
+
+
+#### Remove Mission Prompt After Landing {#resume_mission_prompt}
+
+You will be prompted to remove the mission from the vehicle after the mission completes and the vehicle lands and disarms. 
+This is meant to prevent issues where stale missions are unknowingly left on a vehicle, potentially resulting in unexpected behavior.
+
+### Display Video {#video_switcher}
+
+*QGroundControl* supports RTP and RTSP video streaming over your vehicle's UDP connection.
```

Comment:
QGC supports several video stream options:

RTP/RTSP
Bare UDP
Taisync
Local video devices (like a usb fpv video receiver)

Reply:
Add TCP-MPEG2 and MPEG-TS to the list.
Also note that UVC is only supported on desktop platforms (no Android or iOS)

Reply:
@dogmaphobic @jaxxzer Thanks. I have replaced this with the following. Can you sanity check (ie I don't know anything about most of these, so not sure if they are correctly described in context. This is also why I omitted bare UDP - that doesn't make sense to me as a video stream option. 

----

*QGroundControl* supports a number of video streaming options:
- RTP and RTSP video streaming over your vehicle's UDP connection
- Taisync
- TCP-MPEG2 and MPEG-TS
- USB video devices (UVC) on desktop platforms (Android or iOS not supported).
  This includes webcams and USB FPV video receivers.

Reply:
Actually, please ignore that. This is not the right section to talk about what streaming we support. If needed we should update https://docs.qgroundcontrol.com/en/SettingsView/General.html#video